packages/@react-stately/calendar/src/useRangeCalendarState.ts

```diff
@@ -31,10 +31,15 @@ export interface RangeCalendarStateOptions<T extends DateValue = DateValue> exte
    */
   createCalendar: (name: string) => Calendar,
   /**
-   * The amount of days that will be displayed at once. This affects how pagination works.
+   * The amount of days that will be displayed at once. This affects how pagination works in case visibleDuration is not provided.
    * @default {months: 1}
    */
-  visibleDuration?: DateDuration
+  visibleDuration?: DateDuration,
+  /**
+   * The amount of days that will be used for advancing to next/previous page.
+   * @default visibleDuration
+   */
+  pageDuration?: DateDuration
```

Comment:
I think we'd like to restrict this a bit more to avoid getting in weird cases where `pageDuration` and `visibleDuration` are of different units (e.g. months vs weeks) or where `pageDuration` is greater than `visibleDuration`. I cannot think of a use case for this.

Maybe we could make this an option between choosing to advance by one of the visible unit, or by the visible page size. For example, `pageBehavior="single | visible"`. Not sure if this is a good name/options for the prop but if you have suggestions let us know.

Reply:
Yeah, I totally agree. I am not happy with the naming either, but I could not come up with anything better yet.

Can you take another look? Is that what you had on your mind?